### ISSUE TITLE
Essay Model Answers

### ISSUE THREAD

**Issue by cmaexamacademy** [nickdavis/cmaexamacademy.com#39]
@nickdavis in our essay scenarios, only MCQs show an answer explanation. Could we add answer explanations to the essay questions too? Currently we can only add the question and that's it. 

In this answer explanations, we would show students how they should answer an essay. We would call them Model Answers.

**Comment by nickdavis** [comment]
Yes, we can use the same 50 / 50 columns (answer in sidebar) approach.

Would need to refactor a little bit, because at the moment Essay Questions don't expect to have answers so the flow is slightly different, but it's absolutely do-able.
